lib/Aniki.pm

```diff
@@ -256,6 +256,50 @@ sub update {
     }
 }
 
+sub update_and_fetch_row {
+    my $self = shift;
+
+    if (blessed $_[0] && $_[0]->isa('Aniki::Row')) {
+        local $Carp::CarpLevel = $Carp::CarpLevel + 1;
+
+        $self->update($_[0], $_[1]);
+
+        my $row = $self->select($_[0]->table_name, $self->_where_row_cond($_[0]->table, $_[0]->row_data), { limit => 1, suppress_result_objects => 1 })->[0];
+        return $row if $self->suppress_row_objects;
```

Comment:
if suppress_row_objects is true, can not pass $row object.
Is my understanding correct?
64b13c451ff651bd9dc8cb9c1d0f7ce932f90914